test/unit/lodash.spec.js

```diff
@@ -0,0 +1,522 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+var Guy = require('./Guy');
+var chance = require('chance').Chance();
+
+describe('Lodash training', function ()
+{
+    describe('Math functions', function () {
+        describe('add', function () {
+
+            var params, capacity, area;
+
+            beforeEach(function () {
```

Comment:
Since you are not changing any provided data you should use before instead beforeEach because it would be faster to run tests in that way.